conf/config-schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "object",
+    "properties": {
+        "root": { "type": "boolean" },
+        "globals": { "type": ["object", "array"] },
+        "global": { "type": ["object", "array"] },
+        "parser": { "type": ["string", "null"] },
+        "env": { "type": "object" },
+        "plugins": { "type": ["array"] },
+        "settings": { "type": "object" },
+        "extends": { "type": ["string", "array"] },
+        "rules": { "type": "object" },
+        "parserOptions": { "type": "object" }
+    },
+    "additionalProperties": false
```

Comment:
> Silently ignoring it could lead to people configuring ecmaFeatures there, which would have no effect. This is exactly what this change wants to prevent.

I agree.